edk2toollib/uefi/edk2/path_utilities.py

```diff
@@ -185,20 +185,22 @@ def PackagePathList(self):
         """List of package paths as strings."""
         return [str(p) for p in self._package_path_list]
 
-    def GetEdk2RelativePathFromAbsolutePath(self, abspath: str):
-        """Given an absolute path return a edk2 path relative to workspace or packagespath.
+    def GetEdk2RelativePathFromAbsolutePath(self, *abspath: str) -> str:
+        """Transforms an abolute path to an edk2 path relative to the workspace or a packages path.
 
         Args:
-            abspath: absolute path to a file or directory. Supports both Windows and Posix style paths
+            *abspath: absolute path to a file or directory. Can be the entire path or parts of the path provided
+                separately. Supports both Windows and Posix style paths
```

Comment:
Since this is a new change to the interface, I think an example of calling with parts would be helpful to people considering using that.

Reply:
Added the following example and verified it display properly in the docs:

```python
abs_path = edk2path.GetAbsolutePathOnThisSystemFromEdk2RelativePath("MdePkg/Include")
abs_path = edk2path.GetAbsolutePathOnThisSystemFromEdk2RelativePath("MdePkg", "Include")
```